tests/Makefile

```diff
@@ -1,12 +1,13 @@
-PATH := ../deps/luajit/usr/local/bin:$(PATH)
+PATH := ../deps/luajit/usr/local/bin:../../deps/luajit/usr/local/bin:$(PATH)
```

Comment:
a) The tests are .sh one-liners; the Lua tools are there to make the tests be short, but aren't themselves the tests.
b) That's basically what's happening, aside from the extension.
c) The ugly bit is the paths within the Lua files to find the directories for the code they need to include; it's more relative-path messiness.